integrationTests/multiShard/relayedTx/relayedTx_test.go

```diff
@@ -188,7 +188,7 @@ func testRelayedTransactionInMultiShardEnvironmentWithSmartContractTX(
 		}
 		time.Sleep(time.Second)
 
-		roundToPropagateMultiShard := int64(25)
+		roundToPropagateMultiShard := int64(40)
```

Comment:
Test was extremely hard to debug.

It seems that it was affected by the new behavior of the cache, that does not return transactions with lower nonces during selection.